src/checks/pokerus.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  description: 'Illegal pokerus strain for Pokémon',
+  field: 'pokerus',
```

Comment:
I think this should generally match the one of the Pokémon field names, e.g. `pokerusStrain`